### PR TITLE
write docs

### PR DESCRIPTION
**Pull request by morethanreal**
@notwaldorf PTAL cc @cdata

### CODE REVIEW

File: paper-dialog-behavior.html [hunk 1]
```diff
@@ -32,11 +27,36 @@
...
+to include `confirmed: true`.
+
+### Styling
+
+The following custom properties and mixins are available for styling.
 
-  Polymer.PaperDialogBehavior = Polymer.IronOverlayBehavior.concat({
+Custom property | Description | Default
+----------------|-------------|----------
+`--paper-dialog-background-color` | Dialog background color                     | `--primary-background-color`
```
Comment (notwaldorf) [inline paper-dialog-behavior.html]: Oh, nice. Do you think we should do this table for all elements? (vs. the css sample in, say `paper-checkbox`)

File: paper-dialog-behavior.html [hunk 2]
```diff
@@ -32,11 +27,36 @@
...
+`--paper-dialog-color`            | Dialog foreground color                     | `--primary-text-color`
+`--paper-dialog`                  | Mixin applied to the dialog                 | `{}`
+`--paper-dialog-title`            | Mixin applied to the title (`<h2>`) element | `{}`
+`--paper-dialog-button-color`     | Button area foreground color                | `--default-primary-color`
+
+### Accessibility
```
Comment (notwaldorf) [inline paper-dialog-behavior.html]: Should we add this for all the elements?

File: paper-dialog-behavior.html [hunk 3]
```diff
@@ -32,11 +27,36 @@
             <paper-button dialog-confirm>Accept</paper-button>
         </div>
     </paper-dialog-impl>
--->
 
-<script>
+`paper-dialog-common.css` provide styles for a header, content area, and an action area for buttons.
```
Comment (notwaldorf) [inline paper-dialog-behavior.html]: nit: provides

File: paper-dialog-behavior.html [hunk 4]
```diff
@@ -32,11 +27,36 @@
...
+Use the `<h2>` tag for the header and the `buttons` class for the action area. You can use the
+`paper-dialog-scrollable` element (in its own repository) if you need a scrolling content area.
+
+Use the `dialog-dismiss` and `dialog-confirm` attributes on interactive controls to close the
+dialog. If the user dismisses the dialog with `dialog-confirm`, the `closingReason` will update
+to include `confirmed: true`.
+
+### Styling
+
+The following custom properties and mixins are available for styling.
```
Comment (notwaldorf) [inline paper-dialog-behavior.html]: maybe end in a `:` instead of a `.`?

**Comment by notwaldorf** [comment]
LGTM and questions

File: paper-dialog-behavior.html (hunk 2 above)
Comment (morethanreal) [inline paper-dialog-behavior.html]: Yes, we should.

File: paper-dialog-behavior.html (hunk 1 above)
Comment (morethanreal) [inline paper-dialog-behavior.html]: Yes, I think it would be better to list the custom properties in the docs, rather than having to go to the source.

**Pull request merged**